lib/rubocop/cop/rails/validation.rb

```diff
@@ -4,6 +4,14 @@ module RuboCop
   module Cop
     module Rails
       # This cop checks for the use of old-style attribute validation macros.
+      #
+      # @example
+      #   # bad
+      #   validates_presence_of :foo
```

Comment:
Ideally you should extend this to cover all of the validations - I think there are around 10 of those.